.circleci/config.yml

```diff
@@ -561,24 +558,24 @@ workflows:
   version: 2
   build:
     jobs:
-      - cpu_tests_py37
       - cpu_tests_py38
       - cpu_tests_py39
-      - gpu_tests_1_8_1:
+      - cpu_tests_py310
+      - gpu_tests_lts:
```

Comment:
It would be helpful to add a comment specifying the pytorch version for `lts` and `stable` somewhere (possibly at the top) in the `config.yml` file.

Reply:
Thanks for the review! I think the version is in the pip install command line anyway.